Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -139,17 +203,49 @@
       "OutputDtmi": "dtmi:com:willowinc:EnteringChilledWaterTemperatureSensor;1"
     },
     {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Chilled_Water_Temperature_Setpoint;1",
+      "InputDtmi": "dtmi:mapped:core:Chilled_Water_Supply_Temperature_Setpoint;1",
       "OutputDtmi": "dtmi:com:willowinc:ChilledWaterTemperatureSetpoint;1"
     },
     {
-      "InputDtmi": "dtmi:org:w3id:rec:Company;1",
-      "OutputDtmi": "dtmi:com:willowinc:Company;1"
+      "InputDtmi": "dtmi:mapped:core:Chilled_Water_Usage_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:Sensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Class;1",
+      "OutputDtmi": "dtmi:com:willowinc:Class;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Close_Command;1",
+      "OutputDtmi": "dtmi:com:willowinc:ValveCloseActuator;1"
```

Comment:
Do we know it's a valve and not a damper, say? dtmi:com:willowinc:CloseActuator;1

<img width="430" alt="image" src="https://github.com/mapped/mapped-willow-mapping/assets/84034746/44bbf1fd-cfeb-40c4-aeef-ddfc239db234">